lib/types.ts

```diff
@@ -1,8 +1,53 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
 
+interface PackageJsonPerson {
+  name: string;
+  email?: string;
+  url?: string;
+}
+
+interface PackageJsonBugs {
+  url?: string;
+  email?: string;
+}
+/**
+ * Based on version 9.6.6
+ */
 export interface PackageJsonObject {
   name: string;
   version: string;
+  description?: string;
+  keywords?: string[];
+  homepage?: string;
+  bugs?: PackageJsonBugs | string;
+  /**
+   * Check https://spdx.org/licenses/ for valid licences
+   */
+  license?: "MIT" | "ISC" | string | "UNLICENSED" | "SEE LICENSE IN <filename>";
```

Comment:
```suggestion
  license?: "MIT" | "ISC" | "UNLICENSED" | string;
```

Just because it's not really a valid option.